ncm-spma/src/main/perl/spma/apt.pm

```diff
@@ -0,0 +1,385 @@
+#${PMpre} NCM::Component::spma::apt${PMpost}
+
+=head1 NAME
+
+NCM::Component::spma::apt - NCM SPMA backend for apt
+
+=head1 SYNOPSIS
+
+This document describes how to control the behaviour of the package manager itself.
+For information on how to manage packages with Quattor, please check
+L<http://quattor.org/documentation/2013/04/05/package-management.html>.
+
+=head1 DESCRIPTION
+
+apt.pm implements an apt backend for ncm-spma, the approach taken is to defer as much work as possible to apt.
+
+A single SPMA run consists of the following steps:
+
+=over
+
+=item
+
+Setup source directory if required
```

Comment:
out of interest, does the pod look different if you put this after the `=item` or in a new paragraph like this?